doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -38,3 +38,119 @@ further documented in the `.mplot3d.axes3d.Axes3D.view_init` API.
+drag the mouse in circles with a handedness opposite to the desired rotation.
+
+A different variety of trackball rotates along the shortest arc on the virtual
+sphere (``mouserotationstyle: arcball``); it is a variation on Ken Shoemake's
+ARCBALL [Shoemake1992]_. Rotating around the viewing direction is straightforward
+with it. Shoemake's original arcball is also available
+(``mouserotationstyle: Shoemake``); it is free of hysteresis, i.e.,
+returning mouse to the original position returns the figure to its original
+orientation, the rotation is independent of the details of the path the mouse
+took. However, Shoemake's arcball rotates at twice the angular rate of the
+mouse movement (it is quite noticeable, especially when adjusting roll).
+So it is a trade-off.
+
+Shoemake's arcball has an abrupt edge; this is remedied in Holroyd's arcball
+(``mouserotationstyle: Holroyd``).
+
+Henriksen et al. [Henriksen2002]_ provide an overview.
+
+In summary:
+
+.. list-table::
+   :width: 100%
+   :widths: 30 20 20 20 35
+
+   * - Style
+     - traditional [1]_
+     - incl. roll [2]_
+     - uniform [3]_
+     - path independent [4]_
+   * - azel
+     - ✔️
+     - ❌
+     - ❌
+     - ✔️
+   * - trackball
+     - ❌
+     - ～
+     - ✔️
+     - ❌
+   * - arcball
+     - ❌
+     - ✔️
+     - ✔️
+     - ❌
+   * - Shoemake
+     - ❌
+     - ✔️
+     - ✔️
+     - ✔️
+   * - Holroyd
+     - ❌
+     - ✔️
+     - ✔️
+     - ✔️
+
+
+.. [1] The way it was historically; this is also MATLAB's style
+.. [2] Mouse controls roll too (not only azimuth and elevation)
+.. [3] Figure reacts the same way to mouse movements, regardless of orientation (no difference between 'poles' and 'equator')
+.. [4] Returning mouse to original position returns figure to original orientation (no hysteresis: rotation is independent of the details of the path the mouse took)
+
+Try it out by adding a file ``matplotlibrc`` to folder ``matplotlib\galleries\examples\mplot3d``,
+with contents::
+
+    axes3d.mouserotationstyle: arcball
+
+(or any of the other styles), and run a suitable example, e.g.::
+
+    python surfaced3d.py
+
+(If eternal compatibility with the horrors of the past is less of a consideration
+for you, then it is likely that you would want to go with ``arcball``, ``Shoemake``,
+or ``Holroyd``.)
+
+The size of the trackball or arcball can be adjusted by setting
+``rcParams.axes3d.trackballsize``, in units of the Axes bounding box;
+i.e., to make the trackball span the whole bounding box, set it to 1.
+A size of ca. 2/3 appears to work reasonably well.
+
+----
+
+.. [Shoemake1992] Ken Shoemake, "ARCBALL: A user interface for specifying
+  three-dimensional rotation using a mouse", in Proceedings of Graphics
+  Interface '92, 1992, pp. 151-156, https://doi.org/10.20380/GI1992.18
+
+.. [Henriksen2002] Knud Henriksen, Jon Sporring, Kasper Hornbæk,
+  "Virtual Trackballs Revisited", in Proceedings of DSAGM'2002:
+  http://www.diku.dk/~kash/papers/DSAGM2002_henriksen.pdf;
```

Comment:
It doesn't work for me any more either - it broke recently. Apparently, it moved to
http://hjemmesider.diku.dk/~kash/papers/DSAGM2002_henriksen.pdf
We could change the link to that; or, rather than playing tag and keeping updating links, we could stay closer to the original, and do something like:
https://web.archive.org/web/20040903081925/http://www.diku.dk/~kash/papers/DSAGM2002_henriksen.pdf
What would you recommend?

Reply:
I think either is appropriate, or the link could be removed altogether and people would have to search for the article. I don't have strong feelings here

Reply:
I'll do ``[pdf](https://web.archive.org/web/20240607102518/http://hjemmesider.diku.dk/~kash/papers/DSAGM2002_henriksen.pdf)`` then;

Reply:
I found a more recent version of the paper, and they refer to it as "Bell's Trackball" rather than "Holroyd's Trackball" in the new one, with a reference to the same page. This seems to be a much more common naming when googling around, and I found [this tack overflow](https://stackoverflow.com/a/4676990/11584125) attributing the method to Gavin Bell, as well as listing this as the view rotation method used in Blender.

https://www.researchgate.net/publication/8329656_Virtual_Trackballs_Revisited

Reply:
> [...] So I think the name of this method should be switched over to "Bell",

Yes, it should, I think so too now.

> I can't track down where else the name Holroyd would have come from.

Me neither; strange how the conference preprint silently changed the attribution in the published paper, but not the citation webpage...

Reply:
It's updated: Holroyd -> Bell.